src/plugins/betterFolders/FolderSideBar.tsx

```diff
@@ -45,6 +45,7 @@ export default ErrorBoundary.wrap(guildsBarProps => {
     // Also display flex otherwise to fix scrolling
     const barStyle = {
         display: isFullscreen ? "none" : "flex",
+        gridArea: "sidebar"
     } as CSSProperties;
```

Comment:
should be better (unless it throws error trol)
```suggestion
    } satisfies CSSProperties;
```